utils/generateAddReceivedAssetKeys.ts

```diff
@@ -25,6 +25,7 @@ export const generateAddReceivedAssetKeys = (
     {
       keyName: 'LSP5ReceivedAssets[]',
       value: [assetAddress],
+      // @ts-ignore
```

Comment:
what was the issue here to ignore ts?

Reply:
There are incomplete type in the erc725js regarding `EncodeDataInput`, I believe `DataInput` should have optional `receivedAssetsLength` and `totalArrayLength` fields. Right @CJ42 @richtera ?